src/main/java/frc/robot/Robot.java

```diff
@@ -119,45 +119,42 @@ public void teleopInit() {
     // });
     // }
 
-    // public Command goodFlip() {
-    // CommandChain retValue = new CommandChain("Good Flip");
+    public Command goodFlip() {
+        CommandChain retValue = new CommandChain("Good Flip");
 
-    // retValue.then(lift.goToAtLeast(LiftSubsystem.Heights.kLiftFlipHeight)).then(maflipulator.crappyFlip());
-    // return retValue;
-    // }
+        retValue.then(lift.goToAtLeast(LiftSubsystem.Heights.kLiftFlipHeight)).then(maflipulator.crappyFlip());
+        return retValue;
+    }
 
-    // public Command stowAndGo() {
-    // CommandChain retValue = new CommandChain("Stow it and go!");
-    // retValue.then(lift.goToHeight(Heights.kFloorLoad),
-    // maflipulator.stowAndGoPosition());
-    // return retValue;
-    // }
+    public Command stowAndGo() {
+        CommandChain retValue = new CommandChain("Stow it and go!");
+        retValue.then(lift.goToHeight(Heights.kFloorLoad), maflipulator.stowAndGoPosition());
+        return retValue;
+    }
 
-    // public Command levelOne() {
-    // CommandChain retValue = new CommandChain("Level one");
+    public Command levelOne() {
```

Comment:
I would change the numbers, like "levelOne" to just say what physical level it actually is "goToLoadingStation" or "goToRocketLow"

Reply:
done; changes command names to "goToRocketHighHatch", etc.